js/components/ProfileQuickView.jsx

```diff
@@ -75,18 +64,17 @@ var ProfileQuickView = React.createClass({
     var imgUrl = strings.DEFAULT_USER_PIC;
     var expertCategories = '';
     var investorCategories = '';
-    if (this.state.currentUser) {
-      profileLink = <Link to="profile" params={{userId: this.state.currentUser._id}}>{this.state.currentUser.username}</Link>;
-      about = this.state.currentUser.about ? this.state.currentUser.about : '';
 
-      if (this.state.currentUser.picture && this.state.currentUser.picture.url) {
-        imgUrl = this.state.currentUser.picture.url;
-      }
+    profileLink = <Link to="profile" params={{userId: this.props.currentUser._id}}>{this.props.currentUser.username}</Link>;
+    about = this.props.currentUser.about ? this.props.currentUser.about : '';
 
-      expertCategories = this.getExpertCategories();
-      investorCategories = this.getInvestorCategories();
+    if (this.props.currentUser.picture && this.props.currentUser.picture.url) {
```

Comment:
Since it's props, we know that it exists. This component doesn't get rendered anymore until this.state.currentUser exists from the HomePage